vaccine/tests/test_vaccine_reg_ussd.py

```diff
@@ -841,7 +841,7 @@ async def test_suburb_search():
     )
     app = Application(u)
     msg = Message(
-        content="western cape",
+        content="9",
```

Comment:
We should still be able to use the label `western cape` here, but with alphabetical sorting, western cape should always be last, so this should be fine

Reply:
I wanted to ensure that the tests reflect the user choice more accurately so that the `value` doesn't slip through rather than the choice.